src/nginx/azext_nginx/aaz/latest/nginx/deployment/_create.py

```diff
@@ -143,6 +148,13 @@ def _build_arguments_schema(cls, *args, **kwargs):
             help={"short-summary": "Optional: Preferred communication email", "long-summary": "Usage --user-profile preferred-email=xyz@abc.com"},
         )
 
+        auto_upgrade_profile = cls._args_schema.auto_upgrade_profile
+        auto_upgrade_profile.upgrade_channel = AAZStrArg(
```

Comment:
is there a way to add some "help" around valid values? If so, can you please add `preview` and `stable`

Reply:
fixed